.travis.yml

```diff
@@ -24,7 +24,7 @@ before_script:
 
 script:
   - echo 'Tested in CI'
```

Comment:
This can be moved. (Seen while working on https://github.com/ome/rOMERO-gateway/pull/14#issuecomment-299179708)